src/regions/AudioRegion.js

```diff
@@ -61,10 +61,10 @@ const Model = types
      *   }
      * }
      * @typedef {Object} AudioRegionResult
-     * @property {number} original_length length of the original audio (s)
+     * @property {number} original_length length of the original audio
```

Comment:
That's a format of the value :)
seconds in this case
I put it for further automatic data generation
I'll return it back, but if you have any idea, how to make it better, I'd be glad to hear!

Reply:
I'll use `(seconds)`

Reply:
perfect, thank you! I didn't realize that's what it was lol